pkg/bgpv1/logger/log.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package logger
+
+import (
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+var (
+	// Log to be used in BGP package. All log messages emitted by this have k/v
+	// `subsys=bgp-control-plane`.
+	Log = logging.DefaultLogger.WithField(logfields.LogSubsys, "bgp-control-plane")
+)
+
+const (
+	// NodeConfig is name of the node config resource
+	NodeConfig = "bgp_node_config"
+
+	// Instance is used as key in the log field.
+	Instance = "bgp_instance"
+
+	// LocalASN is BGP instance AS number
+	LocalASN = "asn"
+
+	// ListenPort is local port of BGP instance
+	ListenPort = "listen_port"
+
+	// RouterID is BGP instance router ID
+	RouterID = "router_id"
+
+	// Peer is used as key for BGP peer in the log field.
+	Peer = "bgp_peer"
+)
```

Comment:
When we have `subsys` tag, it is obvious for users that the log is from BGP, so it's redundant to prefix log fields with `bgp`?

Reply:
ack - make sense